core/trino-main/src/main/java/io/trino/operator/TableWriterOperator.java

```diff
@@ -27,7 +27,6 @@
 import io.trino.spi.Mergeable;
 import io.trino.spi.Page;
 import io.trino.spi.PageBuilder;
-import io.trino.spi.TrinoException;
```

Comment:
> Remove checking non nullability via TableWriterOperator

Add cmt msg -- "This is now redundant as the checks are inserted by the planner"